modules/installation-replacing-control-plane-nodes.adoc

```diff
@@ -0,0 +1,163 @@
++
+[source,terminal]
+----
+$ oc project openshift-etcd
+$ oc delete secret etcd-peer-<node_name>
+$ oc delete secret etcd-serving-<node_name>
+$ oc delete secret etcd-serving-metrics-<node_name>
+----
+
+. Delete resources for the failed node:
+
+.. Delete the BMH object by running the following command:
++
+[source,terminal]
+----
+$ oc project openshift-machine-api
+$ oc delete bmh/<node_name>
+----
+.. Delete the machine object that corresponds to the failed node by running the following command:
++
+[source,terminal]
+----
+$ oc delete machine/<machine_name>
+----
+
+. Recreate the failed host by using the same name and IP address:
+
+.. Redefine the virtual machine or provision replacement hardware by using the saved domain XML or hardware profile:
++
+[source,terminal]
+----
+$ TBD
+----
+.. Create the Baseboard Management Controller (BMC) secret for the replacement host by running the following command: 
++
+[source,terminal]
+----
+$ oc create -f backup/bmc-secret.yaml
+----
+.. Update the BMH manifest with the correct MAC address and UUID.
+
+.. Create the BMH object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f backup/bmh.yaml
+----
+
+.. Create the machine object for the replacement node by running the following command:
++
+[source,terminal]
+----
+$ oc create -f backup/machine.yaml
+----
+
+. Approve pending Certificate Signing Requests (CSRs) so the replacement node can join the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | \
+  xargs oc adm certificate approve
+----
+
+. Restore etcd certificates and configuration:
+
+.. Copy etcd certificates from the survivor node by running the following command:
++
+[source,terminal]
+----
+TBD
+----
+
+.. Place the certificates on the replacement node by running the following command:
++
+[source,terminal]
+----
+TBD
+----
+
+.. Force a redeployment of the etcd operator by running the following command:
++
+[source,terminal]
+----
+$ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$(date --rfc-3339=ns)"'"}}' --type=merge
+----
+
+. Rejoin the replacement node to the pacemaker cluster by running the following command:
++
+[source,terminal]
+----
+$ sudo pcs cluster node remove <NODE_NAME>
+$ sudo pcs cluster node add <NODE_NAME> addr=<NODE_IP> --start --enable
+----
+
+. Delete stale jobs for the failed node by running the following command:
++
+[source,terminal]
+----
+$ oc project openshift-etcd
+$ oc delete job tnf-auth-job-<node_name>
+$ oc delete job tnf-after-setup-job-<node_name>
+----
+
+.Verification
+
+* Verify that the replacement node is in the `Ready` state by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+* Verify that etcd members are healthy by running the following command:
++
+[source,terminal]
+----
+$ oc rsh -n openshift-etcd <etcd-pod> etcdctl member list -w table
```

Comment:
Does this work? Since we remove the etcd container from the etcd-pod, I've never run this like this. It's possible you hit the etcdctl container by default and it works fine. I also do this by running this directly from the etcd pod on the node.
$ sudo podman exec -it etcd etcdctl member list -w table